docs/shared/angular-standalone-tutorial/angular-standalone.md

```diff
@@ -811,6 +811,10 @@ If you have the ESLint plugin installed in your IDE you should immediately see a
 
 Learn more about how to [enforce module boundaries](/core-features/enforce-module-boundaries).
 
+## Migrating to a Monorepo
+
+If you want to move `myngapp` out of the root, you can run the [`convert-to-monorepo` generator](/nx-api/workspace/generators/convert-to-monorepo) or [manually move the configuration files](/recipes/tips-n-tricks/standalone-to-integrated).
```

Comment:
just a little comment here. Can we change the "out of the root" which sounds a bit weird. Someone doesn't really want to move "out of the root" but rather expand their current workspace with an additional application etc and therefore transition to a monorepo.

We should also mention the full monorepo tutorial and link it where we have it already